dapps/tests/contracts/test/another_storage_spec.js

```diff
@@ -13,8 +13,11 @@ config({
       "SimpleStorage": {
         args: [100]
       },
+      // "AnotherStorage": {
+      //   args: ["$SimpleStorage", "embark.eth"]
+      // },
       "AnotherStorage": {
-        args: ["$SimpleStorage", "embark.eth"]
+        args: ["$SimpleStorage", "$SimpleStorage"]
```

Comment:
This looks odd to me. Is injecting two SimpleStorage instances to AnotherStorage going to fix tests?

Reply:
No, that's a patch, because support for `.eth` addresses is currently broken. We need to add back the module support feature, which is a big task, that will need to come in another PR.
In the meantime, I pass two times the same address just for the sake of not modifying the contract